src/Sulu/Bundle/AudienceTargetingBundle/Mapper/TargetGroupMapper.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AudienceTargetingBundle\Mapper;
+
+use Sulu\Bundle\AudienceTargetingBundle\Entity\TargetGroupInterface;
+use Sulu\Bundle\AudienceTargetingBundle\Entity\TargetGroupWebspaceRepositoryInterface;
+
+/**
+ * Service that maps data to target groups.
+ */
+class TargetGroupMapper implements TargetGroupMapperInterface
```

Comment:
it's going to be used for rules and conditions as well. 
But do you really think using Handlers is a more sophisticated way of doing this?